api/api.go

```diff
@@ -129,8 +136,16 @@ func (s *authedRequester) Do(r *Request) *Response {
 
 	theurl := s.baseurl
 	theurl.Path += r.Endpoint
-	theurl.RawQuery = urlParams.Encode()
-	req, err := http.NewRequest("GET", theurl.String(), nil)
+	if r.ReqType == "POST" {
+		r.Body.WriteString(urlParams.Encode())
```

Comment:
Could pull out `urlParams.Encode()` into a variable, then you don't need to call it in both branches.